### PR TITLE
feat: fix a few messed up endpoints

### PR DESCRIPTION
**Pull request by granthill7**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

### CODE REVIEW

File: packages/api/tests/api/auth/get.test.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { get } from '../../../src/api/auth/get';
 describe('auth SLACK', () => {
   it('redirects to correct url for happy path', async () => {
     const fullLink =
-      'https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code&redirect_uri=undefined%2Fapi%2Fauth%2Fcallback%2Fget&client_id=undefined';
+      'https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code&redirect_uri=undefined%2Fapi%2Fauth%2Fcallback%2Fslack%2F&client_id=undefined';
```
Comment (SpencerKaiser) [inline packages/api/tests/api/auth/get.test.ts:6]: Ooooof I didn't realize how janky this was; can you do the following? This _should_ work...


Also what the heck is that random undefined doing in there?? 🤨 
```suggestion
      `https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code&redirect_uri=${encodeURIComponent('undefined/api/auth/callback/slack')}&client_id=undefined`;
```

Also don't merge this suggestion; try it locally first and preferably even break it out as a separate var

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
One small change to a janky part of the initial implementation

File: packages/api/tests/api/auth/get.test.ts (hunk 1 above)
Comment (granthill7) [reply packages/api/tests/api/auth/get.test.ts:6]: the undefined's are for values that normally come from env variables

Comment (granthill7) [reply packages/api/tests/api/auth/get.test.ts]: @SpencerKaiser Fixed!

**Review by SpencerKaiser — APPROVED** [review]

**Pull request merged**